### PR TITLE
Add minor UI improvements

### PR DESCRIPTION
**Pull request by hetfly**

### CODE REVIEW

**Comment by parrot9design** [comment]
Let’s completely remove the decimal places for the BTCB and ETH prices in the Position Details section.

The ETH logo has a white box around it in the Liquidity Pools section – might be an issue with the id?
Screenshot 2023-07-17 at 6 40 58 PM

**Comment by hetfly** [comment]
@parrot9design fixed

**Comment by parrot9design** [comment]
Looks great!

For ICX that's currently unstaking, can we add the time remaining for each request? Thinking something like this:
Screenshot 2023-07-18 at 3 50 08 PM

Also wondering if we can close the wallet when you navigate to a different page in the app, so you don't have to close it manually before carrying on with your next task?

**Comment by hetfly** [comment]
@parrot9design both suggestions are implemented 👍

File: src/app/components/Wallet/wallets/ICXWallet/UnstakePanel.tsx [hunk 1]
```diff
@@ -56,51 +57,63 @@ export default function UnstakePanel({ claimableICX }: UnstakePanelProps) {
...
-              Your ICX will be ready to claim in ~7 days, but it may unstake sooner based on the volume of ICX converted
-              to sICX.
-            </Trans>
-          </Typography>
+      {unstakes.map((unstake, index) => {
+        return (
+          <>
+            {index === 0 && (
+              <Typography mb="4">
+                {t`Your ICX will be ready to claim in ${
```
Comment (parrot9design) [inline src/app/components/Wallet/wallets/ICXWallet/UnstakePanel.tsx:100]:
```suggestion
                {t`The ICX unstaking period is ${
```

File: src/app/components/Wallet/wallets/ICXWallet/UnstakePanel.tsx [hunk 2]
```diff
@@ -56,51 +57,63 @@ export default function UnstakePanel({ claimableICX }: UnstakePanelProps) {
...
+                  icxUnstakingTime ? icxUnstakingTime.toFixed(1) : '~7'
+                } days, but it may unstake sooner based on the volume of ICX
```
Comment (parrot9design) [inline src/app/components/Wallet/wallets/ICXWallet/UnstakePanel.tsx:102]:
```suggestion
                } days, but yours may be ready to claim sooner based on the volume of ICX
```

**Review by parrot9design — APPROVED** [review]
Just added a suggestion for the ICX unstaking message in the wallet so it makes more sense with the new info, but other than that, this looks ready to merge. Thanks for your work on this, @hetfly!

**Pull request merged**